xbmc/video/VideoDatabase.cpp

```diff
@@ -11811,13 +11812,13 @@ int CVideoDatabase::AddVideoVersionType(const std::string& typeVideoVersion,
     if (!m_pDB || !m_pDS)
       return -1;
 
-    m_pDS->query(PrepareSQL("SELECT id, owner FROM videoversiontype WHERE name = '%s'",
+    m_pDS->query(PrepareSQL("SELECT id, owner, itemType FROM videoversiontype WHERE name = '%s'",
```

Comment:
Although unlikely in real life, if the user types a version name for the extra (Standard Edition), then this will find the existing "system" record for the version and update it/use it, instead of looking exclusively at extras?
So maybe `WHERE name = 'new name' AND itemtype = assetType`?
Maybe the 'owner' field should also be factored in this, but I don't have a good mental picture of its usage.

Reply:
Right. Thanks. I will change that.

Reply:
> Maybe the 'owner' field should also be factored in this, but I don't have a good mental picture of its usage.

What I found out so far:

owner = SYSTEM 
=> predefined

owner = AUTO 
=> created without user interaction (during scanning? not sure),
=> On "add extra" (maybe also on "add version", you will not asked for selecting a type, the new extra will be created with an auto-created name. You have to use "Rename" later if you want to assign an existing type. Weird.

owner = USER
=> created by the user

Reply:
OK it shouldn't be necessary to include owner in the WHERE at this time, existing logic seems fine.

Auto is the value I wasn't too sure of. Yes it comes from scraper and the add buttons. Auto names are not displayed in the rename lists, but renaming something to an existing auto name will "upgrade" it to "user" owner, visible in future rename lists. The idea is sound but I didn't inspect the code for deviations.